### PR TITLE
Auto-completion of Tags while filing new complaint

### PR DESCRIPTION
**Pull request by simranmadhok**
While filling new complaints, currently the tags are hard coded into the app.
But with the new API venter/tags, the tags will be fetched at the time of filling of the complaint through a retrofit call.

### CODE REVIEW

File: app/src/main/java/app/insti/ComplaintTag.java [hunk 1]
```diff
@@ -1,6 +1,5 @@
 package app.insti;
-
-/**
+ /**
```
Comment (unstablebrainiac) [inline app/src/main/java/app/insti/ComplaintTag.java]: Whitespace change

File: .idea/misc.xml [hunk 2]
```diff
@@ -24,7 +24,7 @@
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
```
Comment (unstablebrainiac) [inline .idea/misc.xml]: Do not commit this file

File: .idea/assetWizardSettings.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (unstablebrainiac) [inline .idea/assetWizardSettings.xml]: Do not commit this file

File: app/src/main/java/app/insti/fragment/FileComplaintFragment.java [hunk 4]
```diff
@@ -131,6 +131,7 @@
     private ImageButton imageButtonAddTags;
     private Button buttonAnalysis;
     private ImageButton imageActionButton;
+    protected List<Venter.TagUri> t = new ArrayList<>();
```
Comment (unstablebrainiac) [inline app/src/main/java/app/insti/fragment/FileComplaintFragment.java]: Change variable name

File: build.gradle [hunk 5]
```diff
@@ -4,8 +4,9 @@ buildscript {
 
     repositories {
         maven { url 'https://maven.google.com' }
-        jcenter()
+
         google()
+        jcenter()
```
Comment (unstablebrainiac) [inline build.gradle]: This is unnecessary

File: app/src/main/java/app/insti/ComplaintTag.java (hunk 1 above)
Comment (simranmadhok) [reply app/src/main/java/app/insti/ComplaintTag.java]: Should I revert?

**Comment by simranmadhok** [comment]
I have removed the two .idea/ files you had asked me to remove. Please review and also let me know about the whitespace changes.

**Pull request closed without merging**